src/Keboola/StorageApi/DirectAccess.php

```diff
@@ -31,4 +31,19 @@ public function resetPassword($backend)
     {
         return $this->client->apiPost("storage/direct-access/{$backend}/reset-password");
     }
+
+    public function enableBucketDirectAccess($bucketId)
```

Comment:
nemělo by to být `enableForBucket()`? Taky máme createCredentials a ne createDirectaccessCredentials...

Reply:
Upravene. Diky ;)